test/es-module/test-importmap.mjs

```diff
@@ -0,0 +1,203 @@
+// Flags: --expose-internals
+
+import { spawnPromisified } from '../common/index.mjs';
+import fixtures from '../common/fixtures.js';
+import tmpdir from '../common/tmpdir.js';
+import { describe, it } from 'node:test';
+import assert from 'node:assert';
+import path from 'node:path';
+import { execPath } from 'node:process';
+import { pathToFileURL } from 'node:url';
+import { writeFile } from 'node:fs/promises';
+import http from 'node:http';
+import import_map from 'internal/modules/esm/import_map';
+const { ImportMap } = import_map;
+import binding from 'internal/test/binding';
+const { primordials: { SafeMap, JSONStringify } } = binding;
```

Comment:
In general, you don’t need to use primordials in tests.

Reply:
The linter told me otherwise IIRC. Happy to do whatever folks recommend, just wanted to call out that I did this because the linter told me to lol.